core/lib/l1_contract_interface/Cargo.toml

```diff
@@ -20,10 +20,12 @@ codegen = { git = "https://github.com/matter-labs/solidity_plonk_verifier.git",
 zkevm_test_harness_1_3_3 = { package = "zkevm_test_harness", git = "https://github.com/matter-labs/era-zkevm_test_harness.git", branch = "v1.3.3" }
 
 # Used to calculate the kzg commitment and proofs
-zkevm_test_harness_1_4_1 = { package = "zkevm_test_harness", git = "https://github.com/matter-labs/era-zkevm_test_harness.git", branch = "v1.4.1" }
+zkevm_test_harness_1_4_2 = { package = "zkevm_test_harness", git = "https://github.com/matter-labs/era-zkevm_test_harness.git", branch = "v1.4.2" }
 sha2 = "0.10.8"
 sha3 = "0.10.8"
 
+once_cell = "1.19"
+
```

Comment:
unused dep?

Reply:
we use it in commit_batch_info for `load_kzg_settings()`